File: airport-web/src/app/shared/components/admin-layout/shared/components/airplane/shared/interfaces/airplane.ts
import {Flight} from "../../../../../../main-layout/shared/interfaces/flight";

export interface Airplane {
  id?: number,
  name: string,
  numberPlaces: number,
  isBusy: boolean,
  isBought?: boolean,
  isDecommissioned?: boolean,
  flight: Flight
}

File: airport-net/airport-net/Models/Airplane.cs
namespace airport_net.Models
{
    public class Airplane
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int NumberPlaces { get; set; }    
        public bool IsBusy { get; set; }
        public bool IsBought { get; set; }
        public bool IsDecommissioned { get; set; }
        public Flight? Flight { get; set; }
    }
}
